tools.py

```diff
@@ -173,13 +177,24 @@ def csv2json(input_path, gz, group, reset_begin_end, csv_sep, audio_name_pattern
 	)
 	# default is Kontur calls pattern, match example: '198.38-200.38_2.0_0_1582594487.376404.wav'
 
-	def duration(audio_name):
+	def match_begin_end(audio_name):
 		match = audio_name_regex.fullmatch(audio_name)
 		assert match is not None, f'audio_name {audio_name!r} must match {audio_name_regex.pattern}'
 		begin, end = float(match['begin']), float(match['end'])
 		assert begin < end < 10_000, 'sanity check: begin and end must be below 10_000 seconds'
+		return begin, end
+
+	def duration(audio_name):
+		begin, end = match_begin_end(audio_name)
 		return end - begin
 
+	def begin_end(audio_name):
```

Comment:
It seems to be an exact duplicate of `match_begin_end(...)`. What are the considerations for adding this duplicate? Thanks